### PR TITLE
WSL: add steps to launch jupyter from the wsl shell

### PR DESCRIPTION
**Pull request by gmanchon**
This conf allows students to open a Jupyter notebook in their browser from the WSL shell. Previously, students would have to manually copy the returned URL of the notebook and paste it in the browser navigation bar.

Thanks @dmilon for the fix 🙏

### CODE REVIEW

File: WINDOWS.md [hunk 1]
```diff
@@ -1210,6 +1210,50 @@ pip freeze
...
+``` bash
+jupyter notebook --generate-config
+```
+
+⚠️ Please copy the path returned by the previous command.
+
+We will now edit the generated Jupyter configuration file:
+
+``` bash
+code /home/gmanchon/.jupyter/jupyter_notebook_config.py # ⚠️ replace this path with your own
```
Comment (dmilon) [inline WINDOWS.md:1227]:
```suggestion
code $HOME/.jupyter/jupyter_notebook_config.py
```

File: _partials/win_jupyter.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+``` bash
+jupyter notebook --generate-config
+```
+
+⚠️ Please copy the path returned by the previous command.
+
+We will now edit the generated Jupyter configuration file:
+
+``` bash
+code /home/gmanchon/.jupyter/jupyter_notebook_config.py # ⚠️ replace this path with your own
```
Comment (dmilon) [inline _partials/win_jupyter.md:15]:
```suggestion
code $HOME/.jupyter/jupyter_notebook_config.py
```

**Review by dmilon — CHANGES_REQUESTED** [review]
:raised_hands: Just one change :pray:

File: WINDOWS.md (hunk 1 above)
Comment (gmanchon) [reply WINDOWS.md]: @dmilon commit gratefully accepted, it works perfectly 🙏 Also the GitHub action seems to have ran the `ruby build.rb`automatically 👍

**Pull request merged**